platform-operator/internal/deployer/kubernetes/kubernetes.go

```diff
@@ -246,74 +246,6 @@ func (d *KubernetesDeployer) createDeployment(ctx context.Context, component *pl
 	if kubeSpec.ImageSpec != nil {
 		imagePullPolicy = kubeSpec.ImageSpec.ImagePullPolicy
 	}
-	initContainers := []corev1.Container{}
-	if d.EnableClientRegistration {
-		initContainers = append(initContainers, corev1.Container{
-			Name:            "kagenti-client-registration",
-			Image:           "ghcr.io/kagenti/kagenti/client-registration:latest",
-			ImagePullPolicy: corev1.PullPolicy(imagePullPolicy),
-			Resources:       component.Spec.Deployer.Kubernetes.Resources,
-			Env: []corev1.EnvVar{
-				{
-					Name: "KEYCLOAK_URL",
-					ValueFrom: &corev1.EnvVarSource{
-						ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
-							LocalObjectReference: corev1.LocalObjectReference{
-								Name: "environments",
-							},
-							Key:      "KEYCLOAK_URL",
-							Optional: ptr.To(true),
-						},
-					},
-				},
-				{
-					Name: "KEYCLOAK_REALM",
-					ValueFrom: &corev1.EnvVarSource{
-						ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
-							LocalObjectReference: corev1.LocalObjectReference{
-								Name: "environments",
-							},
-							Key: "KEYCLOAK_REALM",
-						},
-					},
-				},
-				{
-					Name: "KEYCLOAK_ADMIN_USERNAME",
-					ValueFrom: &corev1.EnvVarSource{
-						ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
-							LocalObjectReference: corev1.LocalObjectReference{
-								Name: "environments",
-							},
-							Key: "KEYCLOAK_ADMIN_USERNAME",
-						},
-					},
-				},
-				{
-					Name: "KEYCLOAK_ADMIN_PASSWORD",
-					ValueFrom: &corev1.EnvVarSource{
-						ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
-							LocalObjectReference: corev1.LocalObjectReference{
-								Name: "environments",
-							},
-							Key: "KEYCLOAK_ADMIN_PASSWORD",
-						},
-					},
-				},
-				{
-					Name:  "CLIENT_NAME",
-					Value: component.Name,
-				},
-				{
-					Name:  "CLIENT_ID",
-					Value: "spiffe://localtest.me/sa/" + component.Name,
-				},
-				{
-					Name:  "NAMESPACE",
-					Value: namespace,
-				},
-			},
-		})
-	}
```

Comment:
There are no more initContainers

This is because client registration now depends on spiffe-helper and spiffe-helper is a long running process so it cannot be an initContainer.